src/Wallabag/CoreBundle/Controller/EntryController.php

```diff
@@ -52,6 +59,25 @@ public function addEntryFormAction(Request $request)
     /**
      * @param Request $request
      *
+     * @Route("/bookmarklet", name="bookmarklet")
+     *
+     * @return \Symfony\Component\HttpFoundation\Response
+     */
+    public function addEntryViaBookmarklet(Request $request)
+    {
+        $url = $request->get('url');
+        if (filter_var($url, FILTER_VALIDATE_URL)) {
```

Comment:
I'm curious, what happend if you don't do that? You got an exception or sth?
This kind of verification is already done in graby.